versions.tf

```
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.0"
+      version = ">= 3.30.0"
```

```suggestion
      version = ">= 3.0"
```
Think we don't need an version increase here since `terraform-aws-modules/apigateway-v2/aws` already defines it here: https://github.com/terraform-aws-modules/terraform-aws-apigateway-v2/blob/4199dd2f1193c895c49b55159b9a08665d8a0947/versions.tf#L5

Terraform is able to check the version requirements of submodules, so it should be able to calculate the correct version.

---

Also applies to the changes in the Readme